package.json

```diff
@@ -14,6 +14,10 @@
     "build": "shx rm -rf dest && npm run fetch && run-p build:images build:js build:pug build:sass",
     "build-uncompressed": "shx rm -rf dest && npm run fetch && run-p build:images build:js-uncompressed build:pug build:sass",
     "fetch": "node scripts/fetch-json.js",
+    "optimize:svg": "find source/images -type f -name '*.svg' -print0 | xargs -0 -n 1 -P 6 svgo --multipass --pretty --enable=removeTitle",
```

Comment:
Ah since this command isn't essential to running the codebase, let's merge it in for now. If Windows support is required, we can refactor; This is already an improvement on what we had.